<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AngleFormulaDialog</class>
 <widget class="QDialog" name="AngleFormulaDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>588</width>
    <height>147</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Formula for right angle</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>110</y>
     <width>341</width>
     <height>32</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>13</y>
     <width>351</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Formula for right angle, dependent of left angle `l`:</string>
   </property>
   <property name="textFormat">
    <enum>Qt::MarkdownText</enum>
   </property>
  </widget>
  <widget class="QComboBox" name="cmbFormula">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>10</y>
     <width>221</width>
     <height>25</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Monospace</family>
    </font>
   </property>
   <property name="editable">
    <bool>true</bool>
   </property>
   <property name="currentText">
    <string>-l</string>
   </property>
   <item>
    <property name="text">
     <string>-l</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>-2*l</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>-l-10</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>50</y>
     <width>151</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Example calculation:</string>
   </property>
   <property name="textFormat">
    <enum>Qt::MarkdownText</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>80</y>
     <width>41</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>`l =`</string>
   </property>
   <property name="textFormat">
    <enum>Qt::MarkdownText</enum>
   </property>
  </widget>
  <widget class="QLineEdit" name="txtExampleLeftAngle">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>78</y>
     <width>61</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>45</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>80</y>
     <width>41</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>`r =`</string>
   </property>
   <property name="textFormat">
    <enum>Qt::MarkdownText</enum>
   </property>
  </widget>
  <widget class="QLabel" name="lblExampleRightAngle">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>80</y>
     <width>411</width>
     <height>21</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(211, 215, 207);
border: 1px solid black;</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="textFormat">
    <enum>Qt::MarkdownText</enum>
   </property>
  </widget>
  <widget class="QPushButton" name="cmdDeleteFormula">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>40</y>
     <width>151</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Delete formula</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>AngleFormulaDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>AngleFormulaDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
